source/particle/world.cc

```diff
@@ -1234,7 +1231,7 @@ namespace aspect
                                "particle weight is recommended. Before adding the weights "
                                "of particles, each cell already carries a weight of 1000 to "
                                "account for the cost of field-based computations.");
-            prm.declare_entry ("Update ghost particles", "false",
+            prm.declare_entry ("Update ghost particles", "true",
```

Comment:
I don't think we have formalized a way to do this yet, but would you mind putting the word "deprecated" as a comment and/or inside the documentation? This way we can search for deprecated parameters after the release.